config.js

```diff
@@ -19,6 +19,9 @@ module.exports = {
         "token_secret": process.env.ACCESS_TOKEN_SECRET
     },
     "twitterKeyWords" : 'pokemon catch,pokemon saw,pokemon attack,pokemon find,pokemon caught,pokemon attacked,pokemon found,pokemon appeared,#foundPokemon,#caughtPokemon,#pokemongo,a wild appeared until #pokemongo',
+    "pokesniperUrl" : "https://pokesnipers.com/api/v1/pokemon.json?referrer=home",
+    "pokesniperListeningInterval" : 10000, // 10 minutes in milliseconds,
+    "pokemonSpawnTime" : 900 ,// 15 minutes in seconds
```

Comment:
It would be more readable if these details(which are specific to pokesniper) are nested under a 'pokesniper' key.